apps/src/templates/teacherNavigation/UnitResourcesDropdown.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import {ActionDropdown} from '@cdo/apps/componentLibrary/dropdown';
+import i18n from '@cdo/locale';
+
+import {RESOURCE_TYPE} from './ResourceIconType';
+
+type UnitResourcesDropdownProps = {
+  unitNumber: number;
+  scriptOverviewPdfUrl: string;
+  scriptResourcesPdfUrl: string;
+};
+
+const UnitResourcesDropdown: React.FC<UnitResourcesDropdownProps> = ({
+  unitNumber,
+  scriptOverviewPdfUrl,
+  scriptResourcesPdfUrl,
+}) => {
+  const dropdownOptions = [
+    {
+      value: 'download-lesson-plans',
+      label: i18n.downloadUnitLessonPlans({unitNumber: unitNumber}),
+      icon: {iconName: RESOURCE_TYPE.LESSON_PLAN.icon},
+      onClick: () => {
+        window.location.href = scriptOverviewPdfUrl;
```

Comment:
In general I would prefer to use [useNavigate](https://reactrouter.com/en/main/hooks/use-navigate) and `navigate` when we are in a react-router context, even if we are not going to a page in the router. What do you think?

Reply:
@kakiha11 I think this could be used for the button we were going to start working on today too.

Reply:
Note to self - since this just downloads the resource and doesn't actually navigate to a different page, we will keep this as is but should use navigate and useNavigate if it is an internal page.